### PR TITLE
Helm InfluxDB Chart

### PR DESCRIPTION
**Pull request by aj-jester**

### CODE REVIEW

File: common/Makefile.default [hunk 1]
```diff
@@ -1,6 +1,10 @@
 SHELL := /bin/bash -o pipefail
 
+ifeq ($(REPLICATED_APP),)
```
Comment (dexhorthy) [inline common/Makefile.default:3]: nice, I like this update

**Comment by dexhorthy** [comment]
@aj-jester is this still WIP or ready to go? When I run `make release` I get this: 


```
deps/replicated release lint --app kots-dex --yaml-dir manifests
Error: read yaml dir: walk manifests: lstat manifests: no such file or directory
```

**Review by dexhorthy — APPROVED** [review]
@aj-jester this looks good, does it need a GH actions workflow? I know we're still working with @austinchambers to get a good overview of what each kotsapp should have and how we want to organize them, so I'm okay with merging and then patching this back up later if there are changes that need to be made to pull this example inline. (just for the sake of staying unblocked) Cool with you @austinchambers ?

**Comment by aj-jester** [comment]
@dexhorthy `make release` will not work here, try `make -C kots helm-release` from the repo root.

**Comment by aj-jester** [comment]
@dexhorthy We can add a GH action sure, the only difference is the command for this is `make -C kots helm-release`, so which ever is the best way y'all think this should be updated we can do that.

**Comment by dexhorthy** [comment]
@aj-jester this doesn't seem to have a kots folder

**Pull request closed without merging**